pkg/skaffold/deploy/kubectl/cli.go

```diff
@@ -18,15 +18,26 @@ package kubectl
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"io"
+	"strings"
+	"time"
 
 	"github.com/sirupsen/logrus"
 
 	pkgkubectl "github.com/GoogleContainerTools/skaffold/pkg/skaffold/kubectl"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
 )
 
+var WaitDeletion = struct {
+	MaxRetry int
+	Delay    time.Duration
+}{
+	MaxRetry: 30,
+	Delay:    2 * time.Second,
```

Comment:
is this something a user would ever want to configure? `maxDeletionWaitTime` or something?

Reply:
probably, yes. I can add that in a second PR